### PR TITLE
Add Verification Code Code Implementation Example

### PR DESCRIPTION
**Pull request by xinzhang-toast**
Partner verification code implementation example

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -8,52 +8,29 @@
...
+      "active": true,
+      "verificationCode": "123"
     },
     {
       "number": "3",
-      "balance": "0.00",
-      "active": false
+      "balance": "500.00",
+      "active": true,
+      "verificationCode": "123"
```
Comment (konradgorski-toast) [inline db.json]: Can we avoid changing the default activation state for cards `3` and `5`, and remove the items from transaction list?

File: cards.js [hunk 2]
```diff
@@ -86,4 +89,19 @@ function update(card) {
   return card;
 }
 
+function validateVerificationCode(verificationCode, identifier, card) {
+  if (verificationCode['source'] === 'VERIFIED' || identifier['source'] === 'SWIPE'
```
Comment (konradgorski-toast) [inline cards.js]: Why would we skip verification for `SWIPE`? With this implementation we should probably assume the setting "Skip verification on swipes" to be off

Comment (xinzhang-toast) [reply cards.js]: updated

File: db.json (hunk 1 above)
Comment (xinzhang-toast) [reply db.json]: updated

**Review by konradgorski-toast — APPROVED** [review]
In PR description - instead of "3rd party", let's refer to "Partner"

**Pull request merged**